app/views/characters/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_for [@house, @character] do |f| %>
+<%= f.label :name %>
+<%= f.text_field :name %>
+<%= f.label :title %>
+<%= f.text_field :title %>
+<%= f.label :gender %>
+<%= f.text_field :gender %>
+<%= f.label :born %>
+<%= f.text_field :born %>
+<%= f.label :died %>
+<%= f.text_field :died %>
+<%= f.label :alias %>
+<%= f.text_field :alias %>
+<%= f.submit "Save" %>
+<% end %>
```

Comment:
I like how leaving a house is implicitly death. If you wanted to be able to change houses for a character, check out `select` which you can use as a method with the form iterator ie `f.select`